smartcontractkit/smart-contract-db
# refactor: updates Courses and Tutorials pages

# Overview
Updates Courses and Tutorials pages

# Screenshots for Mobile and Desktop views (if applicable)
Updated Tutorials:
![image](https://user-images.githubusercontent.com/20880360/137355113-02acc331-1354-4198-83c8-11c5085c39ce.png)

Updated Courses:
![image](https://user-images.githubusercontent.com/20880360/137355182-179171a9-ad4f-4180-8ea7-1f731833d599.png)


### Manual Testing

Write the steps required for how might test this. Example:

1. Run `npm run dev`
2. In browser, goto `localhost:3000/`
3. Navigate to Tutorials and Courses and verify data
4. Get results :tada:
